TypeCobol.Test/Parser/Programs/Cobol85/ContinuationLine/ContinuationLine2.rdz.cbl

```diff
@@ -2,7 +2,7 @@
        PROGRAM-ID. MyPGM2.
        procedure division.
       * KO
-           display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+               display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
@efr15 This file must be reverted. It's already ok. 
And it will cause a conflict with PR #2571

Reply:
Reverted.